_includes/pagination.html

```diff
@@ -17,9 +17,9 @@
           {% else %}
           <a class="Pagination-item" href=
             {% if page == 1%}
-              "/"
+              "{{ site.github.url }}/"
             {% else %}
-              "{{ site.paginate_path | replace: ':num', page }}"
+              "{{ site.paginate_path | replace: ':num', page | prefix site.github.url }}"
```

Comment:
I _believe_ this should be `| prepend: site.github.url`?

https://shopify.github.io/liquid/filters/prepend/